ajax/libs/primer-markdown/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "primer-markdown",
+  "description": "GitHub stylesheets for rendering markdown.",
+  "filename": "build/index.js",
```

Comment:
Let's use `index.js`
Because the basePath is `build`, we don't have to write `build` again.
Thanks